src/widgets/buttons.ts

```diff
@@ -724,6 +730,59 @@ export function renderAsRow (dom: HTMLDocument, pred: NamedNode, obj: NamedNode,
   return tr
 }
 
+/**
+ * A Div to represent a draggable person, etc in a list
+ *
+ * pred is unused param at the moment
```

Comment:
**issue** unused things should not be present

Reply:
This was also from the previous, but since I'm making a whole new function I think it makes sense to take it out you're right.